pulp_smash/pulp3/bindings.py

```diff
@@ -36,7 +36,7 @@ class PulpTaskError(Exception):
 
     def __init__(self, task):
         """Provide task info to exception."""
-        description = task.error["description"]
+        description = task.to_dict()["error"]["description"]
```

Comment:
I'm not opposed to this change, but is it needed?

Reply:
It *looks* like it shouldn't be necessary, but for some reason it was.  I actually made this change back in December in response to some weird corner case where it wasn't working as expected, but I've forgotten what it was exactly.

I'm not opposed to leaving it out until we re-discover the reason though.

Reply:
both would be fine with me.

Reply:
Let's just merge as-is then.